test/jdk/javax/swing/text/html/parser/Parser/8078268/bug8078268.java

```diff
@@ -38,7 +38,8 @@
    @run main bug8078268
 */
 public class bug8078268 {
-    private static final long TIMEOUT = 10_000;
+    private static final float timeoutFactor = Float.parseFloat(System.getProperty("test.timeout.factor", "1.0"));
```

Comment:
```suggestion
    private static final float timeoutFactor =
            Float.parseFloat(System.getProperty("test.timeout.factor", "1.0"));
```
The line is 115 chars long, I suggest wrapping it. Using static import for `parseFloat` will also reduce the length; `getProperty` could also be statically imported.